dashboards/grafana/kubevirt-control-plane.json

```diff
@@ -1,3087 +1,3640 @@
-                "show": true,
-                "sort": "max",
-                "sortDesc": true,
-                "total": false,
-                "values": true
-            },
-            "lines": true,
-            "linewidth": 2,
-            "nullPointMode": "null",
-            "options": {
-                "alertThreshold": true
-            },
-            "percentage": false,
-            "pluginVersion": "7.5.4",
-            "pointradius": 2,
-            "points": false,
-            "renderer": "flot",
-            "seriesOverrides": [],
-            "spaceLength": 10,
-            "stack": false,
-            "steppedLine": false,
-            "targets": [
-                {
-                    "exemplar": true,
-                    "expr": "sum(rate(go_gc_duration_seconds_sum{}[10m])) by (container) / sum(rate(go_gc_duration_seconds_count{}[10m])) by (container)",
-                    "interval": "",
-                    "legendFormat": "{{container}}",
-                    "queryType": "randomWalk",
-                    "refId": "A"
-                }
-            ],
-            "thresholds": [],
-            "timeFrom": null,
-            "timeRegions": [],
-            "timeShift": null,
-            "title": "GC Duration Mean",
-            "tooltip": {
-                "shared": true,
-                "sort": 0,
-                "value_type": "individual"
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${datasource}",
+          "description": "How many seconds has the longest running processor for workqueue been running.",
+          "fill": 8,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 8,
+            "x": 16,
+            "y": 20
+          },
+          "hiddenSeries": false,
+          "id": 26,
+          "legend": {
+            "alignAsTable": true,
+            "avg": false,
+            "current": false,
+            "hideEmpty": true,
+            "hideZero": true,
+            "max": true,
+            "min": false,
+            "rightSide": true,
+            "show": true,
+            "sort": "max",
+            "sortDesc": true,
+            "total": false,
+            "values": true
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "8.2.3",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": true,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "sum(rate(kubevirt_workqueue_longest_running_processor_seconds{job=~\".*kubevirt.*\", instance=~\"$instance\"}[5m])) by (instance, name)",
```

Comment:
[This metrics](https://github.com/kubevirt/kubevirt/blob/19501a76808082f8b95eda818182580f76694b73/pkg/monitoring/workqueue/prometheus/prometheus.go#L109) is a Gauge metric and, as we discussed before, should not be used with `rate()` or `increase()` in the promQL.
Could you please also update this query?

Reply:
Sure, updated!

*Before*:
![image](https://user-images.githubusercontent.com/17428696/143446738-ee55e027-9a0e-4125-bca8-c207df5b62a9.png)

*After*:
![image](https://user-images.githubusercontent.com/17428696/143446808-0b6fa8dc-f07e-444f-a923-79ac655e76ad.png)